pkg/runtime/controller.go

```diff
@@ -84,10 +84,10 @@ type Controller struct {
 }
 
 // RulesKind defines the config kind name of mixer rules.
-const RulesKind = "mixer-rule"
+const RulesKind = "rule"
 
 // AttributeManifestKind define the config kind name of attribute manifests.
-const AttributeManifestKind = "attribute-manifest"
+const AttributeManifestKind = "attribute"
```

Comment:
is `attribute` preferred to `manifest` ?

Reply:
I think so. Manifest is just a collection, the attribute is the important part.